### ISSUE TITLE
RuntimeWarning: coroutine 'ClientSession.close' was never awaited/Unclosed client session

### ISSUE THREAD

**Issue by kazuoteramoto** [rndusr/stig#55]
This is related to #54. Using Arch Linux aiohttp 3.0.1.

This happen with either the v0.8.2a or on c28fd6b, log from c28fd6b is:

```
❯ stig add *.torrent      
Torrent already exists: ***
/usr/lib/python3.6/site-packages/stig/client/aiotransmission/rpc.py:71: RuntimeWarning: coroutine 'ClientSession.close' was never awaited
Unclosed client session
client_session: <aiohttp.client.ClientSession object at 0x7f2accf7ba58>
Unclosed connector
connections: ['[(<aiohttp.client_proto.ResponseHandler object at 0x7f2acb71f9e8>, 362141.441238972)]']
connector: <aiohttp.connector.TCPConnector object at 0x7f2accf7b940>
```

I don't understand asyncio very well, but I think that there is two bugs here. I can 'fix' (???) the RuntimeWarning by [using](https://stackoverflow.com/a/45211192)
```
--- i/stig/client/aiotransmission/rpc.py
+++ w/stig/client/aiotransmission/rpc.py
@@ -68,7 +68,7 @@ class TransmissionRPC():
     def __del__(self, _warnings=warnings):
         if self._session is not None and not self._session.closed:
             _warnings.warn('disconnect() wasn\'t called', ResourceWarning)
-            self._session.close()
+            asyncio.ensure_future(self._session.close())
 
     def on(self, signal, callback, autoremove=True):
         """
```

But I still get the `Unclosed client session`/`Unclosed connector`.

**Comment by rndusr** [comment]
This seems to happen only when a command fails?

It's probably caused at the end of stig.main. When a command fails, sys.exit(1)
is called, the TransmissionRPC instance is collected and ensure_future() starts
a new task. But nobody awaits it and there is nothing else to do, so the process
terminates with the unfinished task.

If all commands run gracefully, TransmissionRPC.disconnect() is called
explicitly, closing the ClientSession, making __del__ a no-op and leaving no
unfinished tasks.

I think I've fixed this and the other #54-related issue, but if someone would
pull the fix/remove-aiohttp.Timeout branch and confirm I'd feel better about
making a bugfix release for the bugfix release.

**Comment by kazuoteramoto** [comment]
> This seems to happen only when a command fails?

Just tested and yes this seems to be correct.

I just pulled `fix/remove-aiohttp.Timeout`@8502e78 and it indeed fixes this bug. Running something like `stig add doesntexist.torrent` now have a clean exit (with status 1).

Thanks!

**Comment by rndusr** [comment]
Thank you. I'll make another bugfix release later.

**Comment by rndusr** [comment]
0.8.3a is out and should fix this and #56.

Thanks again for your help.
